session.go

```diff
@@ -83,6 +83,12 @@ type Session interface {
 	// the request handling loop. Registering nil will unregister the channel.
 	// During the time that no channel is registered, breaks are ignored.
 	Break(c chan<- bool)
+
+	// DisablePTYEmulation disables the default PTY emulation provided by the session
+	// when no external PTY management is present.
+	// Currently (2022-03-12) the only mechanic implemented is NL-to-CRNL translation (`\n`=>`\r\n`).
+	// Note: this is not thread-safe to call with an ongoing call to Write.
```

Comment:
Drop "Note:" and drop "thread-safe".

// A call of DisablePTYEmulation must precede any call to Write.

Or something.

Reply:
added an error on the return path if it has already been written to.

Reply:
(used a channel with a sync.Once closer)